manifests/artifactory.pp

```diff
@@ -1,58 +1,86 @@
 # Define: archive::artifactory
 # ============================
 #
-# archive wrapper for downloading files from artifactory
+# @summary Archive wrapper for downloading files from artifactory
 #
-# Parameters
-# ----------
+# @param [Stdlib::HTTPUrl] url
+#   artifactory download URL
+# @param [Array] headers
+#   HTTP header(s) to pass to source
+#   Default value: []
+# @param [String] path
+#   absolute path for the download file (or use archive_path and only supply filename)
+#   Default value: namevar
```

Comment:
remove defaults and data types. puppet-strings documents those automatically.

Reply:
fixed, thanks for the help!